tests/vmi_multus_test.go

```diff
@@ -55,39 +55,26 @@ var _ = Describe("Multus", func() {
 	var detachedVMI *v1.VirtualMachineInstance
 	var vmiOne *v1.VirtualMachineInstance
 	var vmiTwo *v1.VirtualMachineInstance
-	var nodeAffinity *k8sv1.Affinity
+	var nodes *k8sv1.NodeList
 
 	createVMI := func(interfaces []v1.Interface, networks []v1.Network) *v1.VirtualMachineInstance {
```

Comment:
Please change 'createVMI' function name to fit the new behavior.

Reply:
done